docs/api/modulus.models.rst

```diff
@@ -41,6 +41,9 @@ Currently available models include:
    * - AdaptiveFourierNeuralOperator
```

Comment:
Yes, please update this to AFNO

Reply:
Fixed in [f849071](https://github.com/NVIDIA/modulus/pull/547/commits/f849071daffda7c4c8c055e007e59c42d9c21ed5).